readme.md

```diff
@@ -75,6 +75,8 @@
 - [daktilo](https://github.com/kronik3r/daktilo) - A Jekyll theme with a minimal design inspired by typewriters. by [@kronik3r](https://github.com/kronik3r)
 - [v-odometer](https://github.com/JefferyHus/v-odometer) - Vue.js component to smoothly transition numbers with ease. by [@jefferyhus](https://github.com/JefferyHus)
 - [es6-crawler-detect](https://github.com/JefferyHus/es6-crawler-detect) - ES6 module to help you detected and block bots, crawlers and spiders. by [@jefferyhus](https://github.com/JefferyHus)
+- [vue-dropify](https://www.npmjs.com/package/vue-dropify) - Dropify build for vuejs framework. by [@khofaai](https://github.com/khofaai)
```

Comment:
can you use GitHub repo URLs instead of npm package url .

Reply:
Done,